.circleci/config.yml

```diff
@@ -90,33 +83,19 @@ jobs:
           name: Running PHPUnit
           working_directory: *PLUGIN_PATH
           command: |
-            composer remove --dev phpunit/phpunit && composer install && npm ci && npm run build
+            composer remove --dev phpunit/phpunit && composer install
+            npm ci && npm run build
             composer require --dev phpunit/phpunit 5.7.9
             WP_TESTS_DIR=~/project/wordpress-develop/tests/phpunit ./vendor/bin/phpunit
 
   js-tests:
     executor: php
     steps:
       - checkout
-      - conditionally-skip:
-          directories: js tests/js
```

Comment:
Ported from https://github.com/studiopress/genesis-custom-blocks/pull/41/files#r517919253. That was merged into the editor feature branch.